### DISCUSSION TITLE
Is it possible to invalidate all access tokens?

### DISCUSSION THREAD

**Discussion by hughhhh** [mmohades/Venmo#58] [Q&A] [answered]
Currently trying to invalidate all tokens that have been created. Also if there is a way to see the list of access tokens currently in flight that will work as well.

**Comment by mmohades** [comment, marked as answer]
Changing your password will result in invalidating all the tokens. Also, checkout the "trusted devices" section in your Venmo app and remove all the devices.
I don't think there's a way to see all the tokens.
